src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` or `@BeforeAll` to initialize your `RegistrationServiceImpl` instance. `@BeforeEach` creates a new instance before each test, ensuring test isolation, while `@BeforeAll` creates a single instance for all tests, which could be more efficient if the instance doesn't need to be reset. Choose the one that fits the test scenario.